src/components/simple-color-picker/simple-color-picker.pw.tsx

```diff
@@ -0,0 +1,810 @@
+import React from "react";
+import { HooksConfig } from "../../../playwright";
+import {
+  commonDataElementInputPreview,
+  getDataElementByValue,
+} from "../../../playwright/components";
+import {
+  simpleColor,
+  simpleColorPickerComponent,
+  simpleColorPickerInput,
+  simpleColorDiv,
+} from "../../../playwright/components/advanced-color-picker";
+import { simpleColorPickerLegend } from "../../../playwright/components/simple-color-picker";
+import { CHARACTERS, VALIDATION } from "../../../playwright/support/constants";
+import {
+  checkAccessibility,
+  containsClass,
+  getStyle,
+  verifyRequiredAsteriskForLegend,
+} from "../../../playwright/support/helper";
+import {
+  SimpleColorPickerProps,
+  SimpleColorProps,
+} from "../../../src/components/simple-color-picker";
+import {
+  SimpleColorCustom,
+  SimpleColorPickerCustom,
+} from "../../../src/components/simple-color-picker/components.test-pw";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const testPropValue = CHARACTERS.STANDARD;
+
+const colors = [
+  {
+    color: "#FFFFFF",
+    label: "transparent",
+  },
+  {
+    color: "#0073C1",
+    label: "blue",
+  },
+  {
+    color: "#582C83",
+    label: "purple",
+  },
+  {
+    color: "#E96400",
+    label: "orange",
+  },
+  {
+    color: "#99ADB6",
+    label: "gray",
+  },
+  {
+    color: "#C7384F",
+    label: "flush mahogany",
+  },
+  {
+    color: "#004500",
+    label: "dark green",
+  },
+  {
+    color: "#FFB500",
+    label: "yellow",
+  },
+  {
+    color: "#335C6D",
+    label: "dark blue",
+  },
+  {
+    color: "#00DC00",
+    label: "light blue",
+  },
+];
+
+const indexes = Array.from({
+  length: colors.length,
+}).map((_, index) => index);
+
+test.describe("When SimpleColorPicker is focused", () => {
+  test("should have the expected styling when the focusRedesignOptOut is false", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<SimpleColorPickerCustom />);
+
+    await simpleColorPickerInput(page, 0).focus();
+
+    const focusedColor = simpleColorDiv(page, 0);
+    await expect(focusedColor).toHaveCSS(
+      "box-shadow",
+      "rgba(0, 0, 0, 0.9) 0px 0px 0px 3px inset, rgb(255, 188, 25) 0px 0px 0px 6px inset"
+    );
+    await expect(focusedColor).toHaveCSS(
+      "outline",
+      "rgba(0, 0, 0, 0) solid 3px"
+    );
+  });
+
+  test("should have the expected styling when the focusRedesignOptOut is true", async ({
+    mount,
+    page,
+  }) => {
+    await mount<HooksConfig>(<SimpleColorPickerCustom />, {
+      hooksConfig: {
+        focusRedesignOptOut: true,
+      },
+    });
+
+    await simpleColorPickerInput(page, 0).focus();
+
+    const focusedColor = simpleColorDiv(page, 0);
+
+    await expect(focusedColor).toHaveCSS(
+      "box-shadow",
+      "rgb(255, 255, 255) 0px 0px 0px 2px inset"
+    );
+  });
+});
+
+test.describe(
+  "Should render SimpleColorPicker component and check functionality",
```

Comment:
```suggestion
  "check functionality for SimpleColorPicker component",
```
**(non-blocking suggestion)**: As the below test descriptions all have words "should render", feel a bit duplicated here.